content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -305,7 +305,7 @@ performance_exporters: PerformanceExporterPrometheus
 performance_exporters: PerformanceExporterKafka
 ```
 
-To enable delfin to push alerts to Alertmanager change the value of `alert_exporters`
+To enable delfin to push alerts to alert manager change the value of `alert_exporters`
```

Comment:
This is Alertmanager, the alert management tool part of Prometheus. 
Please change back to Alertmanager